src/Kernel/Attributes/ParseEntity/Models/Requests/IFindParseEntitiesRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace LT.DigitalOffice.Kernel.Middlewares.ParseEntities.Models.Requests
+{
+    public interface IFindParseEntitiesRequest
+    {
+        static object CreateObj()
+        {
+            return new();
+        }
```

Comment:
why not use boolean var for ensure that message service need in entities properties?